src/components/fabSpeedDial/fabSpeedDial.scss

```diff
@@ -0,0 +1,167 @@
+md-fab-speed-dial {
+  $custom-duration: $swift-ease-in-duration / 2;
+  $custom-transition: all $custom-duration $swift-ease-in-timing-function !default;
+
+  $trigger-width: 56px;
+  $trigger-height: 52px;
+  $trigger-offset: 8px;
+
+  display: flex;
+  align-items: center;
+
+  &.md-opened {
+    &.fling {
+      md-fab-actions {
+        .md-fab-action-item {
+          visibility: visible;
+          transform: translateY(0px) !important;
+        }
+      }
+    }
```

Comment:
Can you try to restructure these rules to use less nesting? The nesting makes the rules raise the specificity and is a bit slower.